### PR TITLE
add more status fields to walletinfo

### LINKED ISSUE #555: need to adjust walletinfo RPC for stake pool safety check
The stake pool ensures that the result of the walletinfo RPC has "stakemining": true in it when starting up to ensure that the wallets are properly configured.

enablevoting causes "stakemining" to be set to false where as the deprecated enablestakemining sets it to true.

This incompatibility needs to be rectified before enablestakemining can be properly deprecated and removed.

### PR DESCRIPTION
**Pull request by jolan**
Closes #555.

This adds vote bits/version to the walletinfo result.

### CODE REVIEW

File: rpctest/rpcharness_test.go [hunk 1]
```diff
@@ -2123,13 +2123,6 @@ func testGetSetTicketMaxPrice(r *Harness, t *testing.T) {
 	if err != nil {
 		t.Fatal("SetTicketMaxPrice failed:", err)
 	}
-
```
Comment (alexlyp) [inline rpctest/rpcharness_test.go]: Should revert these changes too for the other PR for stakemining removal.

**Comment by alexlyp** [comment]
Just that one more revert, otherwise looks good

File: rpctest/rpcharness_test.go (hunk 1 above)
Comment (jolan) [inline rpctest/rpcharness_test.go]: I pushed right before you committed the change in dcrd:

https://github.com/decred/dcrd/commit/10adc627c8e04c8888eb9af9fa90b5d401013dc0

So it is unfortunately already removed in dcrd and thus is necessary to remove here too.

Comment (alexlyp) [inline rpctest/rpcharness_test.go]: Ah crap.  Ok

**Comment by alexlyp** [comment]
tACK
